docs/guides/multi-cluster.md

```diff
@@ -0,0 +1,44 @@
+# Recommended Multi-Cluster Architecture
+
+Here is a recommended multi-cluster architecture if you'd like to setup kubernetes service-to-service communications across multiple clusters.
+
+Suppose your organization would like to have one "service mesh" that spans many clusters in one aws account, this service mesh should include the following components:
+- One manually created VPC Lattice service network, (it could create by either AWS Console, CLI, CloudFormation, Terraform or any other tools)
+- Create `VpcServiceNetworkAssociations` between VPC Lattice service network and each config cluster's VPC and workload clusters' VPCs
+- Multiple workload cluster(s), that are used to run application workload(s). workload cluster(s) should only have following workloads related kubernetes objects:
+  - Multiple application workload(s) (Pods, Deployments etc.)
+  - Multiple `Service(s)` for application workload(s)
+  - Multiple `ServiceExport(s)`, that export kubernetes application Service(s) to the "config cluster"
+- One extra dedicated "config cluster", which is act as a "service mesh control plane" and it should include following kubernetes objects:
+  - One `Gateway` that has __same name__ as the manually created VPC Lattice service network name
+  - Multiple `ServiceImport(s)`, that reference to kubernetes application services that export from workload cluster(s)
+  - Multiple `HTTPRoute(s)`,`GRPCRoute(s)`, that have rules backendRefs to `ServiceImport(s)` that referring kubernetes application service(s) in workload cluster(s)
+
+
+You can see this similar production use case at Airbnb: [airbnb mullti-cluster setup](https://www.youtube.com/watch?v=1D8lg36ZNHs)
```

Comment:
I think we should remove this, instead we should have good reasons for this recommendation

Reply:
Any idea on what good reasons we could say?